### DISCUSSION TITLE
Use of langString

### DISCUSSION THREAD

**Discussion by mereolog** [edmcouncil/fibo#1824] [General]
Why do we need rdf:langString?

If the reason is to have language-specific strings, why not to use rdf:PlainLiteral as the [OWL spec](https://www.w3.org/TR/2012/REC-owl2-syntax-20121211/#Strings) recommends?
Then we wouldn't need to define rdf:langString as we currently do in: https://github.com/edmcouncil/fibo/blob/a3a23e6c69b4fd9dba93418c8922a550cf513389/FND/AgentsAndPeople/Agents.rdf#L56-L61

**Comment by ElisaKendall** [comment]
@mereolog You will see this in Commons as well.  We are not using it explicitly in FIBO, but this is needed for some applications. As it happens, rdf:PlainLiteral was defined in the OWL specification because there was a gap between what was in RDF 1.0 and the second edition of the XML Schema Datatypes. The OWL 2 working group added it to OWL as a stop gap, but no one has really implemented it.  Since the OWL 2 specification was revised in 2012, there is a new version of RDF, RDF 1.1, published in 2014.  In that version of RDF, they implemented the notion of a language tagged string, or rdf:langString, which is what most triple store and other vendors have implemented.  But it's a gap in OWL 2 and so we've included it in this union in the Text datatype for those that need it.  That approach, using such a union, was recommended by Evren Sirin at Stardog, in fact.

**Reply by oldskeptic** [reply]
rdf:langString is the solution that I see used most often with support with SPARQL queries. rdf:PlainLiteral  opens up new possibilities but has limited implementation support.
